### ISSUE TITLE
Runtime Error: Graph disconnected

### ISSUE THREAD

**Issue by JulFor** [keras-team/keras#10936]
Hi when running the code below I get the error:
_RuntimeError: Graph disconnected: cannot obtain value for tensor Tensor("actions_3:0", shape=(?, 4), dtype=float32) at layer "actions". The following previous layers were accessed without issue: []_ 

The code: 
from keras import layers, models, optimizers
from keras import backend as K

states1 = layers.Input(shape=(self.state_size,), name='states')
        actions1 = layers.Input(shape=(self.action_size,), name='actions')

         # State layer 1 
        state_layer1 = layers.Dense(units=128, kernel_regularizer=layers.regularizers.l2(2e-6))(states1)
        state_layer1 = layers.BatchNormalization()(state_layer1)
        state_layer1 = layers.Activation("relu")(state_layer1)

        #state layer 2
        state_layer1 = layers.Dense(units=64,kernel_regularizer=layers.regularizers.l2(1e-6))(state_layer1)

        # Action layer 1
        action_layer1 = layers.Dense(units=128,kernel_regularizer=layers.regularizers.l2(2e-6))(actions1)
        action_layer1 = layers.BatchNormalization()(action_layer1)
        action_layer1= layers.Activation("relu")(action_layer1)
        
        
        #Action layer 2 
        action_layer1 = layers.Dense(units=64,kernel_regularizer=layers.regularizers.l2(1e-6))(action_layer1) 
        action_layer1= layers.Activation("relu")(action_layer1)
        
        # Add layers for states and actions 
        combined_layer1 = layers.Add()([state_layer1, action_layer1])
        combined_layer1 = layers.Activation('relu')(combined_layer1)

        # Add final output layer to prduce action values (Q values)
        Q_values = layers.Dense(units=64, name='q_values')(combined_layer1)
                                                                                                         
        # Initialize a Keras model
        self.model = models.Model(inputs=states1, outputs=actions1)

**Comment by Dref360** [comment]
You define the output of your network as the Input action1.

action1 is not the result of any operation so there is no graph.
Don't you want something like :
`self.model = models.Model(inputs=[states1, actions1], outputs=Q_values)`
